app/schemas/selectors/cinematic.js

```diff
@@ -93,6 +106,79 @@ const leftCharacter = shotSetup => (shotSetup || {}).leftThangType
  */
 const rightCharacter = shotSetup => (shotSetup || {}).rightThangType
 
+/**
+ * @param {ShotSetup} shotSetup
+ * @returns {Object|undefined} background Object
+ */
+const backgroundArt = shotSetup => (shotSetup || {}).backgroundArt
+
+/**
+ * @param {Object} o Object that may have slug property
+ * @returns {string|undefined}
+ */
+const slug = o => (o || {}).slug
+
+/**
+ * Returns properties required to place a background Lank.
+ * @param {Object} backgroundArt
+ * @returns {Object|undefined} a background object
+ */
+const background = backgroundArt => {
```

Comment:
Naming suggestion: `getBackgroundFromBackgroundArt`

Reply:
I see a naming pattern in this file, sometimes we are using `get` with the name of the function and sometimes not. Feel free to ignore my naming suggestion if it falls under the pattern, but I would like to understand when are we using `get` vs not.

Reply:
tl;dr intention  was  `get` prefix means public. Otherwise use property name.

I think the file could  be organized to make the naming more clear. Or maybe alternative names should be used.
Basically private getters are named by their property names. Whilst the public getter from the `Shot` or  `DialogNode` has the `get` prefix.

With that intention in mind is there a better way to name the private methods? I copied a functional pattern where the private functions can be composed as if using the `.` operator.

E.g.
`shot.shotSetup.leftCharacter` is the same as `compose(shotSetup, leftCharacter)(shot)`.
Thus the functions reflect the attributes (with the additional verification and chaining `undefined`).

Reply:
Thanks for clarifying the naming pattern used in this file. I dont think we need to change that pattern, but reorganizing/adding a comment about it could help make it clearer.

Reply:
I'll document this somewhere and re-organizing selectors as a post merge task.